src/main/java/service/FruitParser.java

```diff
@@ -0,0 +1,22 @@
+package service;
+
+import dto.TransferObject;
+
+public class FruitParser implements Parser {
+    private static final String SEPARATOR = ",";
+    private static final int OPERATOR_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public TransferObject parse(String line) {
+        String[] lineParse = line.split(SEPARATOR);
+        FruitValidator fruitValidator = new FruitValidator();
```

Comment:
let's use abstraction and have Validator as a field that is passed throw the constructor